### ISSUE TITLE
May a bug in your code?

### ISSUE THREAD

**Issue by shoutOutYangJie** [cybertronai/pytorch-lamb#12]
![image](https://user-images.githubusercontent.com/46734514/145744877-f07c5e91-886d-4108-a471-4f28026f58a3.png)

I use your optimizer and train a cross-modality model.  In the last 5 epochs, the loss value starts to increase. I don't know why?
